src/Model/Endpoint.php

```diff
@@ -453,6 +464,22 @@ public function resourceClass($name = null)
     }
 
     /**
+     * Returns the inflect method or sets a new one
+     *
+     * @param null $method
+     *
+     * @return null|string
+     */
+    public function inflect($method = null)
```

Comment:
`inflectionMethod` would be a better name. Currently the method name sounds as if it inflects string.